mythril/support/signatures.py

```diff
@@ -177,13 +177,13 @@ def __getitem__(self, item):
         """
         return self.get(sighash=item)
 
-    def import_from_solidity_source(self, file_path):
+    def import_from_solidity_source(self, file_path, solc_binary="solc", solc_args=None):
         """
         Import Function Signatures from solidity source files
         :param file_path: solidity source code file path
         :return: self
         """
-        self.signatures.update(SignatureDb.get_sigs_from_file(file_path))
+        self.signatures.update(SignatureDb.get_sigs_from_file(file_path, solc_binary, solc_args))
```

Comment:
can you also add the respective args names here, it might be useful for the future.

Reply:
Updated in 6fd8b76900bc2098e6007b126e5eb2995f43b6d1